### PR TITLE
Bug - Fix active mod ref selector

### PR DESCRIPTION
**Pull request by BLoe**
## What does this PR do?

- Fixes a bug in the editor selector for active mod ref

For more information on our expectations for the PR process, see the
[code review principles doc](https://www.notion.so/pixiebrix/Code-Review-Principles-1ce7276b82a84d2a995d55ad85e1310d?pvs=4)

### CODE REVIEW

File: src/pageEditor/store/editor/editorSelectors.ts [hunk 1]
```diff
@@ -95,7 +95,7 @@ export const selectExpandedModId = ({ editor }: EditorRootState) =>
  */
 export const selectActiveModComponentRef = createSelector(
   selectActiveModComponentFormState,
-  selectExpandedModId,
+  selectActiveModId,
```
Comment (twschiller) [inline src/pageEditor/store/editor/editorSelectors.ts:98]: Is this necessary? Should this just be using `selectActiveModComponentFormState`?

File: src/pageEditor/store/editor/editorSelectors.ts [hunk 2]
```diff
@@ -104,7 +104,10 @@ export const selectActiveModComponentRef = createSelector(
 
     return {
       modComponentId: formState.uuid,
-      modId: modId ?? getStandaloneModComponentRuntimeModId(formState.uuid),
+      modId:
+        modId ??
+        formState.modMetadata.id ??
```
Comment (twschiller) [inline src/pageEditor/store/editor/editorSelectors.ts:109]: Does this need to be `formState.modMetadata?.id`?

Comment (BLoe) [reply src/pageEditor/store/editor/editorSelectors.ts:109]: Yep, good call, thanks

**Review by twschiller — APPROVED** [review]

**Comment by twschiller** [comment]
@BLoe I made the change, but there's a couple E2E tests failing

File: src/pageEditor/store/editor/editorSelectors.ts (hunk 1 above)
Comment (twschiller) [reply src/pageEditor/store/editor/editorSelectors.ts]: I updated the code with the change

Comment (BLoe) [reply src/pageEditor/store/editor/editorSelectors.ts]: Yes, sorry, looks like I slightly misinterpreted how this works, it's only for mod components I guess? Not valid to use when a mod is selected, based on the `assertNotNullish` check here.

**Comment by twschiller** [comment]
@BLoe I just had GitHub update from main to get the test supressions

**Pull request merged**